chapters/refereecommands.adoc

```diff
@@ -128,18 +128,34 @@ It can also be used to resume the game when the game had to be stopped and no te
 
 ==== Penalty Kick
 .Definition
-To initiate a penalty kick, the stop command has to be sent and the ball has to be placed on the <<Penalty Mark, penalty mark>> by the human <<Referee, referee>>.
+The procedure of a penalty kick is as follows:
 
-When the penalty command is issued, one attacking robot is allowed to approach but not touch the ball. This robot will be referred to as the kicker. The defending keeper has to touch the goal line. All other robots have to move behind a line parallel to the goal line and 0.4 meters behind the penalty mark. When these constraints are met, the referee may continue with a <<Normal Start, normal start>> command.
+. The ball is placed by the human referee on the <<Penalty Mark, penalty mark>>.
+. When the <<Penalty Kick, penalty>> command is issued
+.. The defending keeper has to move to the goal line and keep touching it
+.. One attacking robot is allowed to approach the ball but not touch the ball
+.. All other robots have to be 1m behind the ball such that they do not interfere the penalty kick procedure at any time.
+. When the <<Normal Start, normal start>> command is issued, the attacker is allowed to <<Ball Manipulation, manipulate the ball>>. The ball has to only move towards the opponent goal, as measured by its x coordinate in the coordinate system of <<Vision, SSL-Vision>>.
```

Comment:
I'm pretty sure that even after this change you may not use multiple attacking robots for this penalty. But I cannot see it stated clearly in this PR.

Reply:
> All other robots have to be 1m behind the ball such that they do not interfere the penalty kick procedure at any time.

Is this not clear enough?

Reply:
I misread `at any time`, just imagining `at the start`. I know where this confusion starts: The text reads like a "first, second, third" where `all other robots` are in the preparation phase. But you're right, the rules still contain that section. It's just me i guess.